super_team_picker/routes/super_team_picker.js

```diff
@@ -0,0 +1,137 @@
+const express = require("express")
+const knex = require("../db/client")
+const router = express.Router()
+
+
+router.get("/new", (req,res) => {
+    res.render("super_team/new", {cohort: false})
+})
+
+router.get("/", (req,res) => {
+    res.render("super_team/home")
+})
+
+router.get("/index", (req,res) => {
+    knex("super_teams")
+    .orderBy('id', "desc")
+    .then(cohorts => {
+    res.render("super_team/index", {cohorts: cohorts})
+    })
+})
+
+router.get("/:id", (req,res) => {
+    knex("super_teams")
+    .where("id", req.params.id)
+    .first()
+    .then(cohort => {
+        if (!cohort) {
+            res.render("super_team/view", {cohort: cohort, memberSplit: false})
+        } else {
+            res.render("super_team/view", {cohort: cohort, memberSplit: false})
+        }
+    })
+})
+
+router.get("/:id/edit", (req,res) => {
+    knex("super_teams")
+    .where("id", req.params.id)
+    .first()
+    .then(cohort => {
+        if (!cohort) {
+            res.send("super_team/edit", {cohort: false})
+        } else {
+            res.render("super_team/edit", {cohort: cohort})
+        }
+    })
+})
+
+router.get("/:id/split_teams", (req,res) => {
+    knex("super_teams")
+    .where("id", req.params.id)
+    .first()
+    .then(cohort => {
+
+        const method = req.query.teamMethod
+        let quantity = parseFloat(req.query.quantity)
+        const members = cohort.members.split(", ")
```

Comment:
You can split by comma followed by an empty space but risky unless you pre-process your input to conform to your logic or use regex so that it splits the team by a comma OR a comma with a space e.g. `cohort.members.split(/[, ]+/)`